stats/dashboard.py

```diff
@@ -1024,7 +1024,7 @@ def is_text_in_element(elementName):
             'contact-info': self.element.findall('contact-info/email'),
             'location': self.element.xpath('location/point/pos|location/name|location/description|location/location-administrative'),
             'location_point_pos': self.element.xpath('location/point/pos'),
-            'sector_dac': self.element.xpath('sector[@vocabulary="{}" or @vocabulary="{}" or not(@vocabulary)]'.format(self._dac_5_code(), self._dac_3_code())),
+            'sector_dac': self.element.xpath('sector[@vocabulary="{}" or @vocabulary="{}" or not(@vocabulary)]'.format(self._dac_5_code(), self._dac_3_code())) or all_and_not_empty([transaction.xpath('sector[@vocabulary="{}" or @vocabulary="{}" or not(@vocabulary)]'.format(self._dac_5_code(), self._dac_3_code())) for transaction in self.element.xpath('transaction')]),
```

Comment:
Should test version in here for transactions, similar to https://github.com/IATI/IATI-Stats/blob/master/stats/dashboard.py#L1013

`self._major_version() != '1' and all_and_not_empty [...]`